staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -819,6 +819,12 @@ func (az *Cloud) updateNodeCaches(prevNode, newNode *v1.Node) {
 		if _, hasExcludeBalancerLabel := prevNode.ObjectMeta.Labels[v1.LabelNodeExcludeBalancers]; hasExcludeBalancerLabel {
 			az.excludeLoadBalancerNodes.Delete(prevNode.ObjectMeta.Name)
```

Comment:
This isn't needed because the authoritative Insert/Delete is done on the newNode down below, right?